src/main.rs

```diff
@@ -1,159 +1,226 @@
         .header("Authorization", &token)
         .send()?;
     match deploymentconfigs_resp.status() {
-        StatusCode::OK => {},
-        _ => return Err(From::from("Error! Could not fetch deployment configs. Is the namespace wrong?")),
+        StatusCode::OK => {}
+        _ => {
+            return Err(From::from(
+                "Error! Could not fetch deployment configs. Is the namespace wrong?",
+            ))
+        }
     }
     let deploymentconfigs_json: DeploymentResponse = deploymentconfigs_resp.json()?;
     // Get all of the deployment dates of all of the deployments.
     let mut last_deployments = Vec::new();
     for config in deploymentconfigs_json.items {
         last_deployments.push(config.metadata.creation_timestamp);
     }
- 
+
     // Query for rolebindings (for the admins of the namespace)
-    let rolebindings_call = format!("https://{}/apis/authorization.openshift.io/v1/namespaces/{}/rolebindings", endpoint, namespace);
-    let mut rolebindings_resp = client.get(&rolebindings_call)
+    let rolebindings_call = format!(
+        "https://{}/apis/authorization.openshift.io/v1/namespaces/{}/rolebindings",
+        endpoint, namespace
+    );
+    let mut rolebindings_resp = client
+        .get(&rolebindings_call)
         .header("Authorization", &token)
         .send()?;
-    
+
     match rolebindings_resp.status() {
-        StatusCode::OK => {},
-        _ => return Err(From::from("Error! Could not fetch rolebindings for deployment. Is the namespace wrong?")),
-    } 
+        StatusCode::OK => {}
+        _ => {
+            return Err(From::from(
+                "Error! Could not fetch rolebindings for deployment. Is the namespace wrong?",
+            ))
+        }
+    }
     let rolebindings_resp_as_json: RolebindingsResponse = rolebindings_resp.json()?;
-    let rolebindings_json_vector: Vec<String> = rolebindings_resp_as_json.items.into_iter()
+    let rolebindings_json_vector: Vec<String> = rolebindings_resp_as_json
+        .items
+        .into_iter()
         .filter(|item| item.metadata.name == "admin")
         .filter_map(|item| item.user_names)
         .flatten()
         .collect();
 
-    // Build the API response 
-    let api_response = DBItem{
+    // Build the API response
+    let api_response = DBItem {
         name: namespace,
         admins: rolebindings_json_vector,
-        last_deployment: last_deployments.first().unwrap().to_string()
+        last_deployment: last_deployments.first().unwrap().to_string(),
     };
     Ok(api_response)
 }
 
-
-fn get_db_namespace_table() -> Result<Vec<DBItem>, Box<std::error::Error>> {
+fn get_db_namespace_table() -> Result<Vec<DBItem>> {
     // Direct connection to a server. Will not look for other servers in the topology.
-    let client = Client::connect(&env::var("DB_ADDR")?, env::var("DB_PORT")?.to_string().parse::<u16>().unwrap())
-        .expect("Failed to initialize client.");
+    let client = mongodb::Client::connect(
```

Comment:
Notice that here you didn't have a fully-qualified name, even though the name `Client` is used by more than one crate. So you'll see that I removed `Client` from the top-level `use mongodb::{Client, ...}` and instead use the fully-qualified name `mongodb::Client` here.